draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -307,6 +314,15 @@ confirmed messages.
   settings and a network random nonce used to derive the session keys.
 * Data
 
+## Unicast and multicast technology
+
+LoRaWAN technology supports unicast for uplink and downlink communications, but
+also multicast for downlink data: a packet send over LoRaWAN radio link can be
+received by several devices.  It can be useful, for example, in case of large
+binary transfer like firmware upgrade over the air.
```

Comment:
I would add the reference to Remote Multicast Setup Protocol here instead of in the downlink fragmentation section.

Reply:
Agreed